### PR TITLE
Image to links

### LINKED ISSUE #6: Make images clickable and have them open the business’ website in a new tab

### PR DESCRIPTION
**Pull request by adgai19**
Pull request to try and resolve #6 .Currently the url  points to the yelp review site but I hope to get that resolved to make it point at business url

### CODE REVIEW

**Comment by adgai19** [comment]
sorry for the mess with commits. I forgot to pull the changes into upstream before starting to work.

File: src/components/Business/Business.js [hunk 1]
```diff
@@ -3,16 +3,19 @@ import "./Business.css";
 
 class Business extends React.Component {
   render() {
-    const urlstring = `https://www.google.com/maps/search/?api=1&query=${this.props.business.name} ${this.props.business.address}`.replace(/ /g, "+");
+    const addressurl = `https://www.google.com/maps/search/?api=1&query=${this.props.business.name} ${this.props.business.address}`.replace(/ /g, "+");
```
Comment (jvsinghk) [inline src/components/Business/Business.js]: please make it `googleMapAddressUrl` better variables names make other devs understand our code easily

File: src/components/Business/Business.js [hunk 2]
```diff
@@ -3,16 +3,19 @@ import "./Business.css";
...
+    const websiteUrl = this.props.business.url;
```
Comment (jvsinghk) [inline src/components/Business/Business.js]: add `yelpWebsiteUrl` here as we are currently redirecting to the yelp website of the business here

File: src/util/Yelp.js [hunk 3]
```diff
@@ -15,6 +15,7 @@ const Yelp = {
       })
       .then((jsonResponse) => {
         if (jsonResponse.businesses) {
+          console.log(jsonResponse.businesses)
```
Comment (jvsinghk) [inline src/util/Yelp.js]: kindly remove this log from here

File: src/util/Yelp.js [hunk 4]
```diff
@@ -26,6 +27,7 @@ const Yelp = {
             category: business.categories[0].title,
             rating: business.rating,
             reviewCount: business.review_count,
+            url: business.url,
```
Comment (jvsinghk) [inline src/util/Yelp.js:29]: awesome

File: src/components/Business/Business.js [hunk 5]
```diff
@@ -3,16 +3,19 @@ import "./Business.css";
...
     return (
       <div className="Business">
         <div className="image-container">
-          <img src={this.props.business.imageSrc} alt="" />
+          <a href={websiteUrl} target="_blank" rel="noopener noreferrer">
```
Comment (jvsinghk) [inline src/components/Business/Business.js]: Great, use of rel=`noopener` and `noreferrer` here. I appreciate it @adgai19. Thanks for keeping the security in mind.

**Review by jvsinghk — COMMENTED** [review]
Please make the suggested changes and resolve the conversation. We are good to merge.

**Comment by jvsinghk** [comment]
> sorry for the mess with commits. I forgot to pull the changes into upstream before starting to work.

No worries, It happens to all of us at some point in dev life.

**Comment by adgai19** [comment]
> Please make the suggested changes and resolve the conversation. We are good to merge.

Done.

**Review by jvsinghk — APPROVED** [review]
Great @adgai19 Merging it.

**Pull request merged**